gocd/sp.target.gocd.yaml

```diff
@@ -151,3 +167,42 @@ pipelines:
           - staging-bot
         tasks:
           - script: osc -A https://api.suse.de release SUSE:SLE-15-SP5:GA:TEST
+
+  SLE15.SP5.ReleaseScc:
+    parameters:
+      product: sle15sp5
+      target: MyTarget
+      build: "0.0"
+    stages:
+    - Release.Images.Update:
+        approval: manual
+        roles:
+          - SLE
+        resources:
+          - staging-bot
+        tasks:
+          - script: curl --verbose --max-time 3600 http://10.160.0.10:5000/main/update?product=#{product}&target=#{target}&build=#{build}
```

Comment:
Does that work without quotes and the shell acting on `&` and putting the command in the background, ignoring the target= parameter?

Reply:
I have no clue. This is the first pipeline that uses parameters and the YAML plugin and/or gocd documentation is not very extensive on this behaviour.

Reply:
Or here: https://docs.gocd.org/current/configuration/configuration_reference.html#params

Reply:
It's still a shell script afterwards requiring quotes on the argument

Reply:
That's a different URL, but yes - that's what quotes are.

Reply:
Fixed. Please resolve this discussion if this is the way you deem it correct @coolo and/or @DimStar77